### PR TITLE
 Update one_sp_queue_traffic_test.go

### PR DESCRIPTION
**Pull request by saarvind6481**
Pull request from cisco for dp-1.7

### CODE REVIEW

**Review by sezhang2 — CHANGES_REQUESTED** [review]
I am still working on this experimental code. Please wait.

File: feature/experimental/qos/ate_tests/one_sp_queue_traffic_test/one_sp_queue_traffic_test.go [hunk 1]
```diff
@@ -254,7 +258,7 @@ func TestOneSPQueueTraffic(t *testing.T) {
 		"intf2-af2": {
 			frameSize:             1000,
 			trafficRate:           54.1,
-			dscp:                  16,
+			dscp:                  19,
```
Comment (sezhang2) [inline feature/experimental/qos/ate_tests/one_sp_queue_traffic_test/one_sp_queue_traffic_test.go:261]: What is the reason to change the DSCP value from 16 to 19?

**Review by sezhang2 — APPROVED** [review]
Per the discussion, approve the PR as  a workaround.

**Comment by guoshiuan** [comment]
b/271033767 has the internal test results.

**Review by prinikasn — COMMENTED** [review]
/gcbrun

**Review by prinikasn — APPROVED** [review]

**Pull request merged**